expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
```

Comment:
Is GCC a strict requirement, or can it be compiled with, for example, clang?  Any tips on why GCC might be an explicit requirement?